src/apps/AppCenter/AppCenter.js

```diff
@@ -74,14 +74,15 @@ function getExtendedRepos(appInstanceGroups, repos) {
     const appGroup = appInstanceGroups.find(
       appGroup => appGroup.appId === repo.appId
     )
-    const { name, instances, repoName } = appGroup
+    const { name, instances, repoName } = appGroup || {}
+
     return {
       ...repo,
       // Use latest version’s assets
       baseUrl: repoBaseUrl(repo.appId, repo.latestVersion),
-      instances,
-      name,
-      repoName,
+      instances: instances || [],
+      name: name || '',
+      repoName: repoName || '',
```

Comment:
Just wondering, what do you guys think about moving these to be default parameters in the object destructuring @bpierre @rperez89?

Or are there specific cases where we'd be afraid of a `null` or other weird truthy values being present?

Reply:
No I think that would work too!